scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java

```diff
@@ -404,6 +404,35 @@ public void resubmitJob(String jobId) {
         req.send();
     }
 
+    /**
+     * Resubmit a list of jobs
+     *
+     * @param jobIds List of job Ids
+     */
+    public void resubmitAllJobs(List<String> jobIds) {
+        final List<Integer> selectedJobIds = new ArrayList<>(jobIds.size());
+        for (String id : jobIds) {
+            selectedJobIds.add(Integer.parseInt(id));
+        }
```

Comment:
Don ot understand this instruction: Integer.parseInt(id). id is already an Int !!
Also, rather use **list.stream.map** to apply a transformation to each element in the list without creating a new one.

Reply:
id is a string :-)